TombEngine/Objects/Generic/puzzles_keys.cpp

```diff
@@ -333,7 +333,21 @@ void PuzzleDone(ItemInfo* item, short itemNumber)
 	auto triggerIndex = GetTriggerIndex(item);
 
 	if (triggerIndex == 0)
+	{
```

Comment:
Do not copypaste like this. Copypasting code which is literally repeating several lines down the line is a bad habit.